### PR TITLE
fix: reset marketplace button

### PR DESCRIPTION
**Pull request by SunsetTechuila**
the reset button doesn't work right now

### CODE REVIEW

**Comment by theRealPadster** [comment]
This doesn't work? It worked originally.

**Comment by SunsetTechuila** [comment]
Button click event is passed to the function as an argument rn

File: src/logic/Utils.ts [hunk 1]
```diff
@@ -213,17 +213,19 @@ export const generateSortOptions = (t: (key: string) => string) => {
...
 /**
  * Reset Marketplace localStorage keys
  * @param categories The categories to reset. If none provided, reset everything.
  */
-export const resetMarketplace = (...categories: ("extensions" | "snippets" | "theme")[]) => {
+export const resetMarketplace = (categories?: ResetCategory[] | ResetCategory) => {
   console.debug("Resetting Marketplace");
 
   const keysToRemove: string[] = [];
+  const categoriesToReset = Array.isArray(categories) ? categories : (categories ? [categories] : []);
```
Comment (theRealPadster) [inline src/logic/Utils.ts:225]: Is there a reason why this was changed?

Comment (SunsetTechuila) [reply src/logic/Utils.ts]: i don't remember

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**